build/ts/generatePackages.sh

```diff
@@ -0,0 +1,5 @@
+#/bin/sh
+
```

Comment:
Same here btw - would stick in a `mkdir -p /node/esm && mkdir -p /node/cjs` to avoid something from blowing up.

Reply:
👍